airflow/providers/tabular/example_dags/example_tabular.py

```diff
@@ -0,0 +1,46 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from datetime import datetime, timedelta
+
+from airflow import DAG
+from airflow.operators.bash import BashOperator
+from airflow.providers.tabular.hooks.tabular import TabularHook
+
+default_args = {
+    "owner": "airflow",
+    "depends_on_past": False,
+    "email": ["airflow@airflow.com"],
+    "email_on_failure": False,
+    "email_on_retry": False,
+}
+
+bash_command = f"""
+echo "Our token: {TabularHook().get_token_macro()}"
+echo "Also as an environment variable:"
+env | grep TOKEN
+"""
+
+with DAG(
+    "tabular", default_args=default_args, start_date=datetime(2021, 1, 1), schedule_interval=timedelta(1)
```

Comment:
Nit: Possibly we make this named something more unique, even just `tabular_example`? This would be similar to the DBT example dag. Most provider example DAGs seemed to be named the same as the file itself: https://github.com/apache/airflow/blob/df84c4ad42b1bbaf352d14973f1f867c89dc0a3b/airflow/providers/dbt/cloud/example_dags/example_dbt_cloud.py#L33-L39

Reply:
I like that 👍🏻